File: LeetCode/CPP/LeetCode268.cpp
#include <iostream>
#include <vector>

using namespace std;

int missing_num(vector<int>& nums){
    int n = nums.size();
    int max_sum = (n*(n+1))/2;
    for(int num : nums){
        max_sum-=num;
    }
    return max_sum;
}

int main(){
    vector<int> arr = {9,7,6,5,4,3,2,1,0};
    cout<<"Missing Number Is : "<<missing_num(arr)<<endl;
    return 0;
}

File: LeetCode/JAVA/LeetCode268.java
// Given an array nums containing n distinct numbers in the range [0, n],
// return the only number in the range that is missing from the array.

public class LeetCode268{
    public static int missing_Number(int[] arr){
        int n = arr.length;
        int maxSum = (n*(n+1))/2;
        for(int num : arr){
            maxSum-=num;
        }
        return maxSum;
    }
    public static void main(String[] args) {
        int[] arr = {9,6,4,2,3,5,7,0,1};
        System.out.println("Missing Number Is : " + missing_Number(arr));
    }
}